node/WsTokens.js

```diff
@@ -5,14 +5,10 @@ module.exports = class WsTokens {
 
   isValid(token, userId) {
     return new Promise((resolve, reject) => {
-      this.con.query('SELECT count(*) AS count FROM ws_tokens WHERE token = ? AND user_id = ?;',
+      this.con.query('SELECT count(*) AS count FROM `User_Api_Tokens` WHERE `UAT_Token` = ? AND `UAT_User_ID` = ? AND (`UAT_Last_Used` IS NULL || `UAT_Permanent` <> 0);',
```

Comment:
The `UAT_Permanent` is defined as a `TINY INT NOT NULL` so the spaceship operator(`<>`) is prehap's overkill as it should never be null, but ill leave it

Reply:
It's actually a `not equal` operator which is in the [ANSI standard](https://stackoverflow.com/questions/723195/should-i-use-or-for-not-equal-in-t-sql), while `!=` is not, so I didn't use it.

Reply:
I didn't know about the ANSI Standard for it, Ill review my usage of `!=` in the future :smile: